.gitlab-ci.yml

```diff
@@ -27,6 +27,7 @@ staging:
     - ./venv/bin/python -m pip install --no-cache-dir -r requirements.txt
     - make lint
     - export PATH="$PATH:/opt/webdriver"
+    - unset GOOGLE_APPLICATION_CREDENTIALS; export GOOGLE_PROJECT_NAME=firecloud-cgl; export WORKSPACE_NAME=terra-notebook-utils-tests; export GOOGLE_PROJECT_ID=drs-billing-project
```

Comment:
Why is `GOOGLE_APPLICATION_CREDENTIALS` being unset?

Reply:
It doesn't have to be.  I was just being doubly sure.  Will remove.